res/css/structures/_NotificationPanel.scss

```diff
@@ -19,110 +19,93 @@ limitations under the License.
     flex: 1 1 0;
     overflow-y: auto;
     display: flex;
-}
 
-.mx_NotificationPanel .mx_RoomView_messageListWrapper {
-    flex-direction: row;
-    align-items: center;
-    justify-content: center;
-}
+    .mx_RoomView_messageListWrapper {
+        flex-direction: row;
+        align-items: center;
+        justify-content: center;
+    }
 
-.mx_NotificationPanel .mx_RoomView_MessageList {
-    width: 100%;
-}
+    .mx_RoomView_MessageList {
+        width: 100%;
 
-.mx_NotificationPanel .mx_RoomView_MessageList h2 {
-    margin-left: 0px;
-}
+        h2 {
+            margin-left: 0;
+        }
+    }
 
-/* FIXME: rather than having EventTile's default CSS be for MessagePanel,
+    /* FIXME: rather than having EventTile's default CSS be for MessagePanel,
    we should make EventTile a base CSS class and customise it specifically
    for usage in {Message,File,Notification}Panel. */
```

Comment:
Please fix the indentation of these lines

Reply:
Done, thanks.

Reply:
Done, thanks for checking.